conduit/logging.go

```diff
@@ -0,0 +1,20 @@
+package conduit
+
+import (
+	log "github.com/sirupsen/logrus"
+)
+
+// PluginLogFormatter formats the log message with special conduit tags
+type PluginLogFormatter struct {
+	Formatter *log.JSONFormatter
+	Type      string
+	Name      string
+}
+
+// Format allows this to be used as a logrus formatter
+func (f PluginLogFormatter) Format(entry *log.Entry) ([]byte, error) {
```

Comment:
I finally realized that you're implementing the `Format` interface. Neat trick :+1: